### ISSUE TITLE
Add a way to control app's font size

### ISSUE THREAD

**Issue by apippin** [sidneys/pb-for-desktop#34]
The application opens, but the fonts the app are using are quite large. I don't see any settings anywhere to control the font size, to make them smaller throughout the app. This is a feature request to make such font size control possible, or to understand how to change the font size within its current feature set.

## 🏡 Environmental Context

**App Version**
v4.5.2
**Installation Type** 
Custom npm build from git source
**Operating System**
RHEL 6
Enlightenment 17 desktop environment

**Comment by sidneys** [comment]
@apippin 
This feature just landed in version [v5.0.0](../releases).
Please review and share your findings.

**Comment by apippin** [comment]
I've pulled the latest, 5.2.0, from the git repo and built it. I see the window come up with a menu bar that has a View menu with "Zoom Out" and "Zoom In" menu items. I assume this is the mechanism to use to adjust the overall font size? I've tried clicking those options, but no font size changes are happening. Is this the right mechanism to do that? I'm running on RHEL6 with the Enlightenment window manager.
